.github/workflows/ci.yml

```diff
@@ -186,14 +185,14 @@ jobs:
           docker compose up -d relay-1 relay-2 --no-build
           docker compose up -d gateway --no-build
           docker compose up -d client --no-build
-      - name: 'Performance test: ${{ matrix.test_name }}'
+      - name: "Performance test: ${{ matrix.test_name }}"
```

Comment:
Just wanted to confirm that changing this from `'` to `"` isn't going to change any of the expansion/interpolation for the `${{ }}` in the string?

Reply:
Seems to be working fine: https://github.com/firezone/firezone/actions/runs/11804416568/job/32884861190?pr=7322

Reply:
In GitHub expressions there is a difference between `'` and `"`, but in YAML there isn't (AFAIK).

Reply:
According to the spec there is a difference between the two:
https://yaml.org/spec/1.2.2/#single-quoted-style
https://yaml.org/spec/1.2.2/#731-double-quoted-style

I'm guessing it's probably not going to make a difference here, but just wanted to make sure.